ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/HiveAggregateJoinTransposeRule.java

```diff
@@ -224,7 +224,7 @@ public void onMatch(RelOptRuleCall call) {
             }
           }
           side.newInput = relBuilder.push(joinInput)
-              .aggregate(relBuilder.groupKey(belowAggregateKey, null),
+              .aggregate(relBuilder.groupKey(belowAggregateKey, ImmutableList.of(belowAggregateKey)),
```

Comment:
Is it equivalent if we use `relbuilder.groupKey(belowAggregateKey)`?

Reply:
Yes! I will update this.